app/src/main/res/values-de/strings.xml

```diff
@@ -44,29 +44,29 @@
     <string name="notification_settings_title">Benachrichtigung über eingehende Dateien anzeigen</string>
     <string name="overwrite_settings_title">Überschreiben erlauben</string>
     <string name="accept_settings_title">Transfer automatisch akzeptieren</string>
-    <string name="accept_setting_summary_on">Der Transfer wird automatisch akzeptiert</string>
-    <string name="accept_setting_summary_off">Jeder Transfer muss von Dir akzeptiert werden</string>
-    <string name="background_settings_title">Im Hintergrund ausführend</string>
+    <string name="accept_setting_summary_on">Der Transfer wird automatisch angenommen</string>
+    <string name="accept_setting_summary_off">Jeder Transfer muss von Dir bestätigt werden</string>
+    <string name="background_settings_title">Im Hintergrund ausführen</string>
```

Comment:
@uDEV2019 wrote, --> sphenotux comments
Line 47 and 48 say "accepted". You are using two different words to translate. --> Yes.
"akzeptiert" is the right translation for "accepted" (for me). --> Yes, for me as well.
For "bestätigen" it must be originally "confirmed". --> Yes, but the meaning of "bestätigt" is stronger than "akzeptiert" - because what the user must do in this case is to actively accept a confirmation dialogue. Hm...
Leave it as "akzeptiert" or use for both "angenommen". --> Was not aware of how strict one should stick to the English source. Will go for "akzeptiert"

Reply:
Will follow slowscript's suggestion https://github.com/slowscript/warpinator-android/pull/85#issuecomment-987271588 here. Easier to understand for German users IMHO.